### DISCUSSION TITLE
Does spring-server and ktor-server from examples directory support logging?

### DISCUSSION THREAD

**Discussion by sl4wa** [ExpediaGroup/graphql-kotlin#1038] [Q&A] [answered]
Does spring-server and ktor-server from examples directory support logging of performed GraphQL queries? If not, how can it be enabled?

**Comment by smyrick** [comment, marked as answer]
The examples directly don't have some config, but yes both server libraries support logging of incoming requests through different means

* Ktor https://ktor.io/docs/logging.html
* Spring Reactor: https://cloud.spring.io/spring-cloud-gateway/multi/multi__reactor_netty_access_logs.html

**Comment by smyrick** [comment]
Tried to transfer to discussions but it seems GH is having issues. For future library questions, you can start a discussion instead of opening an issue

https://github.com/ExpediaGroup/graphql-kotlin/discussions
